Options.lua

```diff
@@ -1747,9 +1744,15 @@ function opt:updateRaidList(raidID, refreshInfo, refreshAce)
     -- FILL IN ACE OPTIONS
     if refreshAce then
         local raidInfo = self.raids[raidID]
-        local bosses = tree[idString].args
+        local bosses
+        if tree[idString] then
+          bosses = tree[idString].args
+        else
+          -- TODO debug this
```

Comment:
I'm not sure why this is not getting populated, but here are the IDs where the commented-out print did trigger:

```
548
550
531
469
309
564
532
534
509
533
565
249
544
409

-947
-1945
```